Sources/NetworkService/NetworkError/NetworkError+LocalizedError.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension NetworkError: LocalizedError {
+    /// Provides a description of errors that may occur, while the application is running.
+    public var errorDescription: String? {
+        switch self {
+        case .lostConnection:
+            return "Lost connection to the server. Please check your internet connection and try again."
+        case .serverUnavailable:
+            return "The server is currently unavailable. Please try again later."
```

Comment:
In general, the error description should contain the description of the error, not the recommendations on how to handle it:

```suggestion
            return "The server is currently unavailable."
```

Reply:
Thank you. I removed all such recommendations.